sklearn/tests/test_metaestimators.py

```diff
@@ -44,6 +49,38 @@ def __init__(self, name, construct, skip_methods=(),
                                 'predict'])
 ]
 
+TESTED_META = [
+    # pipeline
+    # this fails because tuple is converted to list in fit:
```

Comment:
are these comments still relevant?

Reply:
no because they skip the tests. will rephrase / remove